### DISCUSSION TITLE
How should I install GNU Global?

### DISCUSSION THREAD

**Discussion by elisson-zlq3x** [alefragnani/vscode-language-pascal#149] [Q&A] [answered]
I've found adapted versions of GNU Global like Cygwin, MinGW and one available in Jason Hood's webpage. The URL reference  website to GNU Global points to official website and there's not a Windows' version there.

**Comment by alefragnani** [comment]
Hi @elisson-zlq3x ,

I just discovered the missing windows version link a few weeks ago. I'm not sure the link was to Jason Hood's webpage, but it seems so. I also looked at Chocolatey and it appears to provide GNU Global there, but since I don't use Chocolatey, I'm not sure the package is correct. Maybe winget provides one as well?

I'm not sure I have an older download stored somewhere, because I'm not using Pascal/Delphi as much as before, but I'll take a look, to find out which alternative location the original GNU Global official website was refering to.

Hope this helps

**Reply by sbhoek** [reply]
Hi Alessandro @alefragnani! I happen to have the same problems as @elisson-zlq3x and I tried to find out whether installing by means of Chocolatey would be an option. Unfortunately, that option is no longer working - already for more than 6 years. Check it out here please: https://community.chocolatey.org/packages/global#virus
Is there anything else that you can recommend to us in this respect? Thank you in advance!

**Reply by alefragnani** [reply, marked as answer]
Hi @sbhoek ,

I know it's been a while but, just to let you know, it seems the Windows releases has been bring back to life. I downloaded a new v6.6.12, and it is working just fine. If you are still interested in trying out the extension, that's an opportunity.

Hope this helps

**Comment by elisson-zlq3x** [comment]
My brother had a version of it and then he sent me. Are you still facing that problem?

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
